Assignment1/homeTask.html

```diff
@@ -0,0 +1,83 @@
+                <select>
+                    <option value="" selected>All</option>
+                    <option>International</option>
+                    <option>National</option>
+                    <option>Economics</option>
+                    <option>Sports</option>
+                    <option>Editorial</option>
+                </select>
+            </div>
+            <div class="two">
+                <h2 class="newsHeading">ALL NEWS</h2>
+            </div>
+            <div class = "three">
+                <label>SUBSCRIBE</label><br>
+                <input type="text" placeholder="email address" class="emailBox">
+                <input type="button" class="subscribeButton" value="Subscribe">
+            </div>
+        </div>
+        <hr class="mainDivision">
+        <br>
+        <div class="descriptionSection">
+            <img class="newsImg" src="image.jpg"/>
+            <div class="newsDesc">
+                <span class="newsTitle"><b>Title One</b></span>
+                <br>
+                <span class="newsDate">Posted on <b>29 June,2019</b> // Category: <b>Category One</b></span>
+                <p class="newsDetail">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <input type="button" class="readingButton" value="Continue Reading">
+            </div>
+        </div>
+        <hr class="subDivision">
+        <div class="descriptionSection">
+            <img class="newsImg" src="image.jpg"/>
+            <div class="newsDesc">
+                <span class="newsTitle"><b>Title One</b></span>
+                <br>
+                <span class="newsDate">Posted on <b>29 June,2019</b> // Category: <b>Category One</b></span>
+                <p class="newsDetail">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <input type="button" class="readingButton" value="Continue Reading">
+            </div>
+        </div>
+        <hr class="subDivision">
+        <div class="descriptionSection">
+            <img class="newsImg" src="image.jpg"/>
+            <div class="newsDesc">
+                <span class="newsTitle"><b>Title One</b></span>
+                <br>

```

Comment:
```
<br> or <br/>.
```
 pls check

Reply:
Checked this, actually in HTML5 we don't need closing slash, before we needed it like in XHTML1.0